dev-util/scons/scons-4.3.0.ebuild

```diff
@@ -1,18 +1,18 @@
 # Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=7
+EAPI=8
 
 DISTUTILS_USE_SETUPTOOLS=rdepend
-PYTHON_COMPAT=( python3_{8..9} )
+PYTHON_COMPAT=( python3_{8..10} )
 PYTHON_REQ_USE="threads(+)"
 
-inherit distutils-r1
+inherit distutils-r1 multiprocessing
 
 DESCRIPTION="Extensible Python-based build utility"
 HOMEPAGE="https://www.scons.org/"
 SRC_URI="
-	https://downloads.sourceforge.net/project/scons/scons/${PV}/${P}.tar.gz
+	https://github.com/SCons/scons/archive/refs/tags/${PV}.tar.gz -> ${P}.tar.gz
```

Comment:
Now it's listed twice, see below. Also I'm pretty sure GH archives missed some files, that's why I had to use sf releases.

Reply:
I have tried to use SF but fails to download